src/popup/components/AddressTruncated.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div>
+    <span>{{ truncatedAddress[0] }}</span>
+    <span
+      class="dots"
+      :class="{ 'medium-dots': mediumDots }"
+    >
+      &middot;&middot;&middot;
+    </span>
+    <span>{{ truncatedAddress[1] }}</span>
+  </div>
+</template>
+
+<script>
+import { computed, defineComponent } from '@vue/composition-api';
+import { truncateAddress } from '../utils';
+
+export default defineComponent({
+  props: {
+    address: { type: String, required: true },
+    mediumDots: Boolean,
+  },
+  setup(props) {
+    const truncatedAddress = computed(() => truncateAddress(props.address));
+
+    return {
+      truncatedAddress,
+    };
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@use '../../styles/typography';
+
+.dots {
+  @extend %face-mono-15-regular;
+
+  letter-spacing: -0.25em;
+  text-align: center;
+  margin-left: -1px;
+  margin-right: 3px;
+
+  &.medium {
+    @extend %face-mono-15-medium;
+  }
```

Comment:
I think the difference between the medium and regular is negligible so I suggest to use only one variant.